assets/elm/Data/Board.elm

```diff
@@ -128,8 +155,13 @@ nextSnakeAndApples time snake apples =
 
 
 score : Board -> Int
-score { snake } =
-    List.length snake.body
+score { currentPlayerId, snakes } =
+    case Dict.get currentPlayerId snakes of
+        Nothing ->
+            0
```

Comment:
here it is in the MaybeX package
http://package.elm-lang.org/packages/elm-community/maybe-extra/4.0.0/Maybe-Extra#unwrap

there's actually alot of places that I Maybe.map then Maybe.withDefault, i've gotten used to putting it on two lines. i even break it up like this:
```
    snakes
      |> Dict.get currentPlayerId
      |> Maybe.map (List.length << .body)
      |> Maybe.withDefault 0
```

This way, I start with the source of the data that I want to transform, then step through the transformations. This matches my step by step mental model of what's happening.